orbit/cmd/orbit/orbit.go

```diff
@@ -616,6 +616,31 @@ func main() {
 			Platform:       osqueryHostInfo.Platform,
 		}
 
+		// Get the hardware UUID. We use a temporary RocksDB location in order to guarantee that
```

Comment:
It's osquery itself that uses a RocksDB database (identified by the path we pass to `getHostInfo`), right? We don't use `RocksDB` per se in the fleet codebase?

Reply:
right, exactly!

Reply:
I went ahead and changed it to "osquery" just to avoid any confusion in the future.